### ISSUE TITLE
Crash on Huawei device for 3.3.6+

### ISSUE THREAD

**Issue by nostra13** [helpcrunch/android-sdk-demo#81]
Crash on Huawei P40 lite (JNY-LX1) (Android 10) for 3.3.6+ lib.

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{by.eleven.scooters.dev/com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity}: android.content.res.Resources$NotFoundException: Font resource ID #0x7f09000a could not be retrieved.
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3897)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4076)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2473)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:219)
       at android.app.ActivityThread.main(ActivityThread.java:8349)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)
```
```
Caused by android.content.res.Resources$NotFoundException: Font resource ID #0x7f09000a could not be retrieved.
       at androidx.core.content.res.ResourcesCompat.loadFont(ResourcesCompat.java:570)
       at androidx.core.content.res.ResourcesCompat.getFont(ResourcesCompat.java:411)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.d(SourceFile:331)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.a(SourceFile:1028)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.a(SourceFile:2463)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.a(SourceFile:2451)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.h(SourceFile:7)
       at com.helpcrunch.library.ui.screens.main.HelpCrunchMainActivity.onCreate(SourceFile:16)
       at android.app.Activity.performCreate(Activity.java:8085)
       at android.app.Activity.performCreate(Activity.java:8073)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1320)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3870)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4076)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2473)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:219)
       at android.app.ActivityThread.main(ActivityThread.java:8349)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)
```

**Comment by TalbotGooday** [comment]
Hello! Thanks for reporting! We don't officially support Huawei, so I can't offer any advice on this.

**Comment by TalbotGooday** [comment]
The problem caused by downloadable fonts in notifications. Fixed in 3.3.8
